eval/src/tests/eval/tensor_lambda/tensor_lambda_test.cpp

```diff
@@ -215,16 +215,31 @@ TEST("require that type resolving also include nodes in the inner tensor lambda
     EXPECT_EQUAL(types.get_type(*symbol).to_spec(), "double");
 }
 
+size_t num_exported(const NodeTypes &types) {
```

Comment:
I would prefer something like count_types, since this is not really related to exporting